code-studio/src/js/components/progress/peer_review_progress_dot.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import Radium from 'radium';
+import { connect } from 'react-redux';
+
+import { peerReviewShape } from './types';
+import { createOutline, styles } from './progress_dot_styles';
+
+const PeerReviewProgressDot = React.createClass({
+  propTypes: {
+    peerReview: peerReviewShape,
+    courseOverviewPage: React.PropTypes.bool
+  },
+
+  linkText: function () {
+    if (this.props.peerReview) {
+      if (this.props.peerReview.submitted) {
+        return 'Link to your completed review';
```

Comment:
I18n?  PLC is en_us only, but we've used i18n for other strings to keep them all in one place.